src/test/test-models/good/function-signatures/math/matrix/matrix_exp.stan

```diff
@@ -0,0 +1,26 @@
+data {
+    int d_int;
+    matrix[d_int,d_int] d_matrix;
+}
+
+transformed data {
+	matrix[d_int,d_int] transformed_data_matrix;
+	transformed_data_matrix = matrix_exp(d_matrix);
+}
+
+parameters {
+	real y_p;
+	matrix[d_int,d_int] p_matrix;
+}
+
+transformed parameters {
+	matrix[d_int,d_int] transformed_param_matrix;
+	
+	transformed_param_matrix = matrix_exp(p_matrix);
+	transformed_param_matrix = matrix_exp(d_matrix);
+}
+
+model {
+	y_p ~ normal(0,1);
+}
+	
```

Comment:
I think you want to be putting newlines at the end of all your files.  Otherwise, they come up with red warnings in GitHub.